.pylintrc

```diff
@@ -11,13 +11,10 @@ disable=
   # Warnings present in existing code. These should be fixed and removed from this list.
   cell-var-from-loop,
   dangerous-default-value,
-  f-string-without-interpolation,
```

Comment:
These removals look good, but I looked over the other warnings, and some of them seem like they would also be good to remove (and hence check), e.g., `cell-var-from-loop`, `redefined-builtin`, `unused-variable`, and (eventually) `unused-wildcard-import`

Reply:
These were all ignored because they were present in existing code when we started checking warnings (#371). As noted in the comment in .pylintrc, all of these should eventually be fixed and removed from the ignore list (unless there's a specific reason for ignoring one).

This PR was limited to a few warnings to keep it small and allow for easier/quicker review. Also, trying to fix many unrelated things in one PR risks the entire PR getting held up on discussion of one specific item. With the set of warnings in this PR, that didn't seem likely because all of these warnings were straightforward and only occurred one or a few times.

Reply:
There is not.

Reply:
Filed an issue for it (#378).